examples/batch_dispatching.rs

```diff
@@ -0,0 +1,150 @@
+//! This example show how to use the Batch dispatcher.
+//!
+//! The Batch is a feature that allow to control the dispatching of a group of systems.
+//!
+//! Specifically here we have three Systems
+//! - **SayHelloSystem**: Which is directly registered under the main dispatcher.
+//! - **BuyTomatoSystem** and **BuyPotatoSystem** are registered in the batch.
+//!
+//! Notice that none of them is using any direct dependency.
+//! The SayHelloSystem is requesting the resources **TomatoStore** and **PotatoStore**,
+//! which are also requested by the other two systems inside the batch and by the batch
+//! controller itself.
+//!
+//! This to demonstrate that the dispatcher is able to correctly organize, their execution.
+//!
+//! Is also showed how to use a custom BatchControllerSystem.
+//! Check the **CustomBatchControllerSystem** which execute its inner Systems three times.
+//!
+
+use shred::{
+    AccessorCow, BatchAccessor, BatchBuilder, BatchController, BatchUncheckedWorld, Dispatcher,
+    DispatcherBuilder, Read, RunningTime, System, World, Write,
+};
+use std::thread::sleep;
```

Comment:
Those should go above `shred` imports (the order for imports typically is 1. `std` 2. `some_crate` 3. `crate::` 4. `self::`).

Also, please run `cargo +nightly fmt` so those get nicely grouped together.

Reply:
using cargo nightly fmt put them in this order:
```
use shred::{
    AccessorCow, BatchAccessor, BatchBuilder, BatchController, BatchUncheckedWorld, Dispatcher,
    DispatcherBuilder, Read, RunningTime, System, World, Write,
};
use std::{thread::sleep, time::Duration};
```

is it ok?